.github/workflows/pytest.yml

```diff
@@ -480,9 +262,9 @@ jobs:
           pip list
         shell: bash
 
-      - name: Integration Tests (E)
+      - name: Integration Tests
         run: |
-          RUN_PRIVATE=$IS_NOT_FORK LUDWIG_TEST_SUITE_TIMEOUT_S=7200 pytest -v --timeout 300 --durations 100 -m "not slow and not combinatorial and not horovod and not llm and not integration_tests_a and not integration_tests_b and not integration_tests_c and not integration_tests_d" --junitxml pytest.xml tests/integration_tests
```

Comment:
@justinxzhao why is it that only test suite E had all these `not integration_tests_a ...` markers but the others don't?

Reply:
Ah, this is because tests for groups a/b/c/d, are explicitly marked with `pytest.mark` while the last bucket (in this case "e"), is a catch-all for all of the rest of unmarked tests. The thinking here was that folks adding a new file in `integration_tests/` shouldn't have to think about assigning a marker.